.github/workflows/run_tests.yml

```diff
@@ -35,9 +35,15 @@ jobs:
       run: |
         REDIS_URL=redis://localhost:6379
         echo REDIS_URL=$REDIS_URL >> $GITHUB_ENV
+    - name: 'Authenticate to Google Cloud'
+      uses: 'google-github-actions/auth@v1'
+      with:
+        credentials_json: ${{ secrets.GOOGLE_CREDENTIALS }}
```

Comment:
I think this needs to be read in first. I've never done this before, so a couple things I found

- https://cloud.google.com/blog/products/identity-security/enabling-keyless-authentication-from-github-actions
- https://github.com/VeraZab/nyc-stats/blob/main/.github/workflows/deploy-flow.yaml

Reply:
I tried this approach because the other method with base64 encoding wasn't working.